### DISCUSSION TITLE
FLOWUnsteady rotor blade postprocessing

### DISCUSSION THREAD

**Discussion by shahfiqshahjahan05** [byuflowlab/FLOWUnsteady#96] [General]
I would like to export AoA, Reynolds number, Mach number, Cl and Cd as a function of r/R and could not find any post processing script for this. I am aware that there is a post processing script for exporting Normal/tangential load, circulations etc. I am wondering if there are any quick way where I can export the quantities I mentioned earlier? Thank you!

**Comment by EdoAlvarezR** [comment]
Passing the flag `debug=true` to the function `run_simulation(...; debug=true)` should be telling the rotor solver to output all its intermediate calculations (like cl and cd). Would you mind testing that and seeing if that gives you the information you need?

**Reply by shahfiqshahjahan05** [reply]
Yes, setting the debug flag to true will output the intermediate calculations into the .vtk file. I have written my own post-processing script that will read the .vtk file and export cl, cd etc. for further post-processing.

**Reply by ilm71** [reply]
Hello, would you mind to share your post-processing script reading the .vtk-files directly? Thank you!

**Reply by gabriel-ces** [reply]
I appreciate the information you provided about setting the debug flag to true in the run_simulation function, enabling the output of intermediate calculations into the .vtk file. 

@shahfiqshahjahan05 would you be willing to share your post-processing script for reading the .vtk files and export cl, cd ?

**Reply by dingzhw** [reply]
@ilm71 @gabriel-ces I got it to work with the Python package PyVista. You can try it out.

**Reply by nrnaveenkumr** [reply]
Hi @shahfiqshahjahan05 , it will be helpfull if you can share your post-processing code, Thanks in Advance
